### DISCUSSION TITLE
[feature request] Smart Playlists for Subsonic collection

### DISCUSSION THREAD

**Discussion by bhankas** [strawberrymusicplayer/strawberry#978] [Ideas]
**Describe the bug**
My favorite feature in Strawberry is to simple click Dynamic random mix and let it play. Usually it is pretty good at selecting tracks from a fairly diverse collection and works really well. Due to $LIFE I am on a different machine without my music collection and having to stream it via self-hosted Subsonic compatible streaming server (navidrome). Strawberry supports Subsonic servers from get go, however, the Smart Playlists do not work with Subsonic collection.

**To Reproduce**
1. Enable and connect to subsonic instance with collection.
2. Do not add any local music folders
3. Try playing any smart playlist.

**Expected behavior**
Smart playlists can play collection from Subsonic server.

**Screenshots:**
N/A

**System Information:**
 - Operating system: Windows 11
 - Strawberry Version: 1.0.5 MSVC

**Additional context**
If it is possible at all to support Smart Playlists for Subsonic, it should be thought about how the UI will work when local collection also happens to be present. My personal use is exclusively via subsonic, but if any user has both, Smart Playlist should be able to control which collection to pick tracks from.

I also found this from the forums, but there isn't much information: https://forum.strawberrymusicplayer.org/topic/383/using-smart-playlists-with-subsonic-collection?lang=en-US

**Comment by jonaski** [comment]
Add this to https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
